wpeverest/user-registration
# UR-636 New - Dashboard Page

### All Submissions:






### Changes proposed in this Pull Request:
Previously our plugin didn't have a Dashboard page to show details about our plugin, products, and many more sections. This PR adds this feature.

### How to test the changes in this Pull Request:
1. Switch to the branch and open the terminal and run `npm run start` or `npm run build`. If any error occurs please resolve the issues and run again.
2. Go to User Registration -> Dashboard.
3. Check if every feature inside the Dashboard page is working properly.
4. Check if the Setup Wizard and User Registration Blocks are working properly as the build of this feature may hamper them.

### Types of changes:




### Other information:




### Changelog entry

> New - Dashboard Page.
